README.md

```diff
@@ -57,7 +57,7 @@ jobs:
 
 - `entry_point`: (Optional) Name of a function (as defined in source code) that will be executed. Defaults to the resource name suffix, if not specified.
 
-- `memory_mb`: (Optional) The amount of memory in MB available for a function. Defaults to '256' (256 MB). The value must be the number of megabytes _without_ the unit suffix.
+- `memory_mb`: (Optional) The amount of memory in MB available for a function. Defaults to '256' (256 MB). The value must be the number of megabytes _without_ the unit suffix. [Possible values](https://cloud.google.com/sdk/gcloud/reference/functions/deploy#--memory) are '128', '256', '512', '1024', '2048', '4096', and '8192'.
```

Comment:
I would prefer to just link to the authoritative source instead of duplicating content that might be inaccurate in the future.